hub/executors/encoders/image/tfkeras-mobilenet_v2/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM jinaai/jina:devel
+
+RUN pip install tensorflow
```

Comment:
Following changes will speed up the pulling speed if user has the `tensorflow` image, which is more likely than having the `jina` image. ;)
```
FROM tensorflow/tensorflow:latest
RUN pip install jina[devel]
```